README.md

```diff
@@ -1,3 +1,30 @@
 # Workleap.OpenApi.MSBuild
 
 Validates at build time that the OpenAPI specification files extracted from the ASP.NET Core Web API being built conform to Workleap API guidelines.
+
+Depending if the users the Contract-First or Code-First approach this MsBuild task will:
+
+- Install tools: OasDiff, Spectral, SwashbuckleCLI
```

Comment:
Maybe provide links to the tools?

- https://github.com/Tufin/oasdiff
- https://github.com/stoplightio/spectral
- https://github.com/domaindrivendev/Swashbuckle.AspNetCore?tab=readme-ov-file#swashbuckleaspnetcorecli